powsybl-geo-data-model/src/main/java/infrastructure/BaseVoltage.java

```diff
@@ -0,0 +1,51 @@
+/**
+ * Copyright (c) 2019, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package infrastructure;
+
+import java.awt.Color;
+
+import java.util.Objects;
+
+/**
+ * @author Chamseddine Benhamed <chamseddine.benhamed at rte-france.com>
+ * @author Geoffroy Jamgotchian <geoffroy.jamgotchian at rte-france.com>
+ */
+public enum BaseVoltage {
```

Comment:
This class is related to very specific RTE open data import as other TSOs might not share same base voltages and associated colors. To move to rte open data repo.